docs/lab/README.md

```diff
@@ -0,0 +1,132 @@
+# Kernel Module Management Lab 
+
+Kernel Module Management a.k.a KMM manages out of tree kernel modules in Kubernetes.
+
+By installing the controller in a Kubernetes cluster we will get a new Namespace where a Deployment will run and create a ReplicaSet.
+
+Once the ReplicaSet is running we can create and use a new Module object which will manage the build and/or run of a kernel module.
+
+In this document we will find examples and YAML files including in-cluster module building, device plugins and use of module loaders with [KMM](https://github.com/kubernetes-sigs/kernel-module-management) for different Kubernetes environments : [EKS](#elastic-kubernetes-service-eks), [GKE](#google-kubernetes-engine-gke), [Azure](#microsoft-azure-aks), [Ubuntu bare metal](#ubuntu-bare-metal-cluster). 
```

Comment:
I don't think there is a point to link the main repository - we are in it.

Regarding the other links, I am not sure they are doing what you wanted them to do. There are all pointing to https://github.com/kubernetes-sigs/kernel-module-management/diffs/0?base_sha=8a500997de386b095e3e69212587ed013c78f1a5&commentable=true&head_user=enriquebelarte&name=labv1&pull_number=23&sha1=8a500997de386b095e3e69212587ed013c78f1a5&sha2=1dbc8af8e3730fa155ce33a0a1f629158cf41192&short_path=94cc112&unchanged=expanded&w=false#elastic-kubernetes-service-eks